privacy_bot/find_policies.py

```diff
@@ -131,35 +98,37 @@ def main():
 
     # Fetch data
     if urls:
-        print(">> Privacy Bot",                 file=sys.stderr)
-        print('-' * 80,                         file=sys.stderr)
-        print("Processing %s urls" % len(urls), file=sys.stderr)
-        print("Number of jobs: %s" % jobs,      file=sys.stderr)
-        print('-' * 80,                         file=sys.stderr)
-        print('',                               file=sys.stderr)
+        print('-' * 42, file=sys.stderr)
+        print('{0:35}'.format('Initializing Privacy Bot'), file=sys.stderr)
+        print('-' * 42, file=sys.stderr)
+        print('{0:35}{1:7}'.format('Urls to Process: ', len(urls)), file=sys.stderr)
+        print('{0:35}{1:7}'.format('Number of Jobs: ', jobs), file=sys.stderr)
+        print('-' * 42, file=sys.stderr)
+        print('', file=sys.stderr)
```

Comment:
Are we talking about the indentation or the size of the lines? They are two different things. Also, maybe we should avoid global file refactoring with code changes, it becomes difficult to read.